mne/preprocessing/infomax_.py

```diff
@@ -136,7 +148,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     while step < max_iter:
 
         # shuffle data at each step
-        rng.seed(step)  # --> permutation is fixed but differs at each step
```

Comment:
@dengemann no, the EEGLAB algorithm [`runica.m`](https://sccn.ucsd.edu/svn/software/eeglab/functions/sigprocfunc/runica.m) only sets the seed once at the beginning to a value depending on the clock (line 812).